setup.py

```diff
@@ -346,7 +357,7 @@ def _populate_hdf5_info(dirstosearch, inc_dirs, libs, lib_dirs):
         else:
             if HDF5_incdir is None:
                 HDF5_incdir = os.path.join(HDF5_dir, 'include')
-            hdf5_version = check_hdf5version(HDF5_incdir)
+            hdf5_version = get_hdf5version(HDF5_incdir)
```

Comment:
this should have an underscore
```suggestion
            hdf5_version = get_hdf5_version(HDF5_incdir)
```